src/nimqml/private/status/statusosnotification.nim

```diff
@@ -0,0 +1,14 @@
+proc setup(self: StatusOSNotificationObject) =
+  self.vptr = dos_statusosnotification_create()
+
+proc delete*(self: StatusOSNotificationObject) =
```

Comment:
```suggestion
proc delete*(self: StatusOSNotificationObject) =
  if self.vptr.isNil:
    return
```

Reply:
@richard-ramos why don't we leave everything to `dotherside`? I feel `nimqml` like a bridge between `status-desktop` and `dotherside` parts, we shouldn't have any logic there, just forward/transfer what's needed, it's just a dumb nim api for c++ methods. Maybe once we will be able to debug `dotherside` part, and wonder why `dos_statusosnotification_delete` is not called. :)

Reply:
If this will not fail if it receives a nil pointer i see no problem with keeping logic in dotherside: https://github.com/status-im/dotherside/pull/45/files#diff-78b5ff497d785fbddb8bf59fd940a02e98143a24b215c4692663f10beb7e988fR1378-R1382

The purpose of this suggestion was too make the delete() proc behavior similar to the other delete* implementations that can be find around in nimqml.

Reply:
@richard-ramos fixed it there, thanks.